### PR TITLE
Fix default cart item thumbnail on sidebar

### PR DESCRIPTION
**Pull request by senghe**

### CODE REVIEW

File: packages/theme/components/CartSidebar/ProductItem.vue [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+          "
+          @click="actionsHandler"
+        >
+          <SfIcon icon="more" size="18px" />
+        </SfButton>
+      </template>
+    </slot>
+  </div>
+</template>
+<script>
```
Comment (dead-playmod) [inline packages/theme/components/CartSidebar/ProductItem.vue:97]: script here is made in vue 2 style most components so far were made in vue 3 style

File: packages/theme/components/CartSidebar/ProductItem.vue [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+    hasRemove: {
+      type: Boolean,
+      default: true
+    },
+    hasMoreActions: {
+      type: Boolean,
+      default: true
+    }
+  },
+  data() {
```
Comment (dead-playmod) [inline packages/theme/components/CartSidebar/ProductItem.vue:174]: data, computed and methods can be put in to a setup() function everything above can stay as it is.

File: packages/theme/components/CartSidebar/ProductItem.vue [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+    return {
+      productPlaceholder
```
Comment (dead-playmod) [inline packages/theme/components/CartSidebar/ProductItem.vue:176]: in setup() you can declare this data as const if it needs to be reactive you can wrap it in ref like this `const productPlaceholder = ref(default value here)`  and a the and of setup() return this const end result should look like this:
```js
setup() {
  const productPlaceholder;
  
  return productPlaceholder;
}
```

File: packages/theme/components/CartSidebar/ProductItem.vue [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+    };
+  },
+  computed: {
```
Comment (dead-playmod) [inline packages/theme/components/CartSidebar/ProductItem.vue:179]: computed can be declared inside of a setup() like this `const componentIs = computed(() => {})` computed has to be imported from `@vue/composition-api` computed also has to be returned at the and of the setup() to use props inside of the setup() you have to define the like this `setup(props)` end result should look like this
```js
setup() {
  const componentIs = computed(() => props.link ? 'SfLink' : 'div');
  const quantity = computed(() => typeof this.qty === 'string' ? Number(this.qty) : this.qty);
  
  return {
    componentIs,
    quantity
  };
}
```

File: packages/theme/components/CartSidebar/ProductItem.vue [hunk 5]
```diff
@@ -0,0 +1,199 @@
...
+    componentIs() {
+      return this.link ? 'SfLink' : 'div';
+    },
+    quantity() {
+      return typeof this.qty === 'string' ? Number(this.qty) : this.qty;
+    }
+  },
+  methods: {
```
Comment (dead-playmod) [inline packages/theme/components/CartSidebar/ProductItem.vue:187]: methods can be declared like this `const removeHandler = () => {}` and it also has to be returned

File: packages/theme/components/CartSidebar/ProductItem.vue (hunk 4 above)
Comment (senghe) [reply packages/theme/components/CartSidebar/ProductItem.vue:179]: :ok_hand:

File: packages/theme/components/CartSidebar/ProductItem.vue (hunk 5 above)
Comment (senghe) [reply packages/theme/components/CartSidebar/ProductItem.vue:187]: :ok_hand:

**Pull request merged**